system-addon/content-src/components/MessageCenterAdmin/MessageCenterAdmin.scss

```diff
@@ -20,27 +21,31 @@
 
   .message-item {
     &:first-child td {
-      border-top: 1px solid $black-10;
+      border-top: 1px solid $border-color;
     }
 
     td {
       vertical-align: top;
-      border-bottom: 1px solid $black-10;
+      border-bottom: 1px solid $border-color;
       padding: 8px;
 
       &:first-child {
-        border-left: 1px solid $black-10;
+        border-left: 1px solid $border-color;
       }
 
       &:last-child {
-        border-right: 1px solid $black-10;
+        border-right: 1px solid $border-color;
       }
     }
 
     &.current {
       .message-id span {
         background: $yellow-50;
         padding: 2px 5px;
+
+        .dark-theme & {
```

Comment:
^ what does that do? 🤔

Reply:
Oh yeah sorry – you can see this page by checking out `about:newtab#message-center-admin` – it just makes the label more visible in the admin panel if dark theme is on

Reply:
I didnt know about that trick so that it wraps the whole selector in `.dark-theme {}`. Awesome.

Reply:
Here's the comparison:
![image](https://user-images.githubusercontent.com/1455535/39141613-456b909c-46f6-11e8-8970-48f85d5bf360.png)

![image](https://user-images.githubusercontent.com/1455535/39141604-404476a6-46f6-11e8-9485-3096eeddd038.png)